client/components/telecom/telephony/abbreviatedNumbers/translations/Messages_fr_FR.xml

```diff
@@ -4,7 +4,7 @@
    <translation id="telephony_abbreviated_numbers_write_error" qtlid="334244">Impossible de mettre à jour le numéro abrégé.</translation>
    <translation id="telephony_abbreviated_numbers_write_success" qtlid="334257">Le numéro abrégé {{num}} a été mis à jour.</translation>
    <translation id="telephony_abbreviated_numbers_id" qtlid="334283">Numéro abrégé</translation>
-   <translation id="telephony_abbreviated_numbers_id_help" qtlid="480279">Doit être un nombre à 3 ou 4 chiffres, débutant par un 7.</translation>
+   <translation id="telephony_abbreviated_numbers_id_help" qtlid="480279">Doit être un nombre à 3 ou 4 chiffres, débutant par un {{digit}}.</translation>
```

Comment:
Maybe the `qtlid` should be removed. Can you check if you're supposed to keep it in that case ?

Reply:
This is done :)